pipeline/src/transformers/eventDocument.ts

```diff
@@ -139,7 +139,7 @@ const transformTimes = ({
     .map((time): EventDocumentTime | undefined => {
       return {
         startDateTime: asDate(time.startDateTime) || undefined,
-        endDateTime: asDate(time.endDateTime) || undefined,
+        endDateTime: asDate(time.endDateTime) || new Date(2100, 1, 1),
```

Comment:
Undefined will not work for our range check, so if an event (most likely a permanent exhibition) does not have an end date, we assign one way in the future.